crates/re_space_view/src/data_query_blueprint.rs

```diff
@@ -96,6 +97,64 @@ impl DataQueryBlueprint {
                 .join(&Self::RECURSIVE_OVERRIDES_PREFIX.into()),
         }
     }
+
+    pub fn remove_entity_expr(&self, ctx: &ViewerContext<'_>, expr: EntityPathExpr) {
```

Comment:
the name is a bit bad imho: We're not removing an entity expression. We're adding an exclusion. So `add_entity_exclusion` instead?